File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.go
package main

import (
	"fmt"
	"strconv"
)

func main() {
	var indexValue, index int
	var data string
	for i := 0; i < 3; i++ {
		fmt.Scan(&data)
		value, _ := strconv.Atoi(data)
		if value != 0 {
			indexValue = value
			index = i
		}
	}
	indexValue += 4 - (index + 1)

	if indexValue%3 == 0 && indexValue%5 == 0 {
		fmt.Println("FizzBuzz")
	} else if indexValue%3 == 0 {
		fmt.Println("Fizz")
	} else if indexValue%5 == 0 {
		fmt.Println("Buzz")
	} else {
		fmt.Println(indexValue)
	}
}

File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.py
index_value = 0
index = 0

for i in range(3):
    data = input().strip()
    try:
        value = int(data)
        index_value = value
        index = i
    except ValueError:
        pass

index_value += 4 - (index + 1)

if index_value % 3 == 0 and index_value % 5 == 0:
    print("FizzBuzz")
elif index_value % 3 == 0:
    print("Fizz")
elif index_value % 5 == 0:
    print("Buzz")
else:
    print(index_value)